logstash-core/lib/logstash/codecs/delegator.rb

```diff
@@ -1,7 +1,9 @@
 module LogStash::Codecs
   class Delegator < SimpleDelegator
-    def initialize(obj)
+    def initialize(obj, parent_plugin_name)
```

Comment:
LS' `Delegator` seems to be gaining some more responsibilities ... 
personally do not like it but do not want to block the PR due my preference
// *someone please chime in e.g. @colinsurprenant was looking at this recently*